cmake/bintools/oneApi/target.cmake

```diff
@@ -0,0 +1,17 @@
+# SPDX-License-Identifier: Apache-2.0
+
+set(find_program_clang_args )
+set(find_program_binutils_args )
+
+find_program(CMAKE_AR      llvm-ar      ${find_program_clang_args}   )
+find_program(CMAKE_NM      llvm-nm      ${find_program_clang_args}   )
+find_program(CMAKE_OBJDUMP llvm-objdump ${find_program_clang_args}   )
+find_program(CMAKE_RANLIB  llvm-ranlib  ${find_program_clang_args}   )
+find_program(CMAKE_OBJCOPY objcopy      ${find_program_binutils_args})
+find_program(CMAKE_READELF readelf      ${find_program_binutils_args})
+
```

Comment:
Interresting observation, when I run CMake multiple times I notice the following:
- First time: `CMAKE_STRIP` is undefined, thus running ninja result in: `&& --strip-all zephyr.elf -ozephyr.strip` being executed leading to following error:
  `/bin/sh: 1: --strip-all: not found`
- Second run: Re-running CMake using: `cmake .` gets me this as `CMAKE_STRIP` value: `CMAKE_STRIP-NOTFOUND`
- Third run: Re-running CMake once more: `cmake .` gets me this as `CMAKE_STRIP` value: `/usr/bin/strip`

Reply:
now finding CMAKE_STRIP which is found as `/opt/intel/oneapi/intelpython/latest/bin/llvm-strip`